2.6.md

```diff
@@ -2751,15 +2757,43 @@ Extended identifiers support in the OpenRTB specification allows buyers to use a
     <td><strong>Type&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
     <td><strong>Description</strong></td>
   </tr>
+<tr>
+    <td><code>inserter</code></td>
+    <td>string</td>
+    <td>The canonical domain name of the entity (publisher, publisher monetization company, SSP, Exchange, Header Wrapper, etc) that caused the ID array element to be added. This may be the operational domain of the system, if that is different from the parent corporate domain, to facilitate WHOIS and reverse IP lookups to establish clear ownership of the delegate system.<br>
+    </br>This should be the same value as used to identify sellers in an ads.txt file if one exists.<br>
+</br>For ad tech intermediaries, this would be their domain as used in ads.txt.  For publishers, this would generally correspond to the OWNERDOMAIN or MANAGERDOMAIN.
+</td>
+ </tr>
   <tr>
     <td><code>source</code></td>
     <td>string</td>
-    <td>Source or technology provider responsible for the set of included IDs. Expressed as a top-level domain.</td>
+    <td>Canonical domain of the ID.<br>
+</br><b>EXAMPLES FOR PUBLIC COMMENT PERIOD ONLY</b>
+<li>Examples of a universal ID include “id5.io”, “liveramp.com”, “unifiedid.com”.</li>
+<li>Examples of a device ID include “apple.com”, “samsung.com”, “roku.com”.</li>
+    </td>
+  </tr>
+<tr>
+    <td><code>matcher</code></td>
+    <td>string</td>
+    <td>Technology providing the match method as defined in <code>mm</code>.<br>
+</br>In some cases, this may be the same value as inserter.<br>
+</br>When blank, it is assumed that the <code>matcher</code> is equal to the <code>source</code><br>
+</br>May be omitted when mm=0, 1, or 2.<br>
+    </br><b>EXAMPLES FOR PUBLIC COMMENT ONLY: LiveIntent, Transunion, LiveRamp</b></br>
+</td>
+ </tr>
+<tr>
+    <td><code>mm</code></td>
+    <td>int</td>
+    <td>Match method used by the <code>matcher</code>. Refer to <a href="https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md#list_agenttypes">List: ID Match Methods</a> in AdCOM 1.0
+    </td>
   </tr>
   <tr>
     <td><code>uids</code></td>
     <td>object array</td>
-    <td>Array of extended ID <code>UID</code> objects from the given source. Refer to 3.2.28 Extended Identifier UIDs</td>
+    <td>Array of extended ID <code>UID</code> objects from the given source. Refer to <a href=https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/main/2.6.md#3228---object-uid-"3.2.28"</a> Extended Identifier UIDs</td>
```

Comment:
This URL is broken, and we shouldn't use HTML anchors or hard links of this form in here anyway. If we want to link, let's use a relative link like this `[Extended Identifier UIDs](#objectuid)`.  Though everywhere else we refer to other sections in this document, we seem to just refer to the section (e.g. `<code>Extended Identifier UIDs</code> object (Section 3.2.28)`).

Reply:
fixed...i think